runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -979,6 +979,14 @@ private void updateFrameConfig(boolean updateResizable)
 			return;
 		}
 
+		// Update window opacity iff the frame is undecorated, translucency capable and not fullscreen
```

Comment:
```suggestion
		// Update window opacity if the frame is undecorated, translucency capable and not fullscreen
```

Reply:
`iff` is shorthand for `if and only if` <https://en.wikipedia.org/wiki/If_and_only_if>